index.md

```diff
@@ -57,7 +57,7 @@ Follow the [Getting Started with JanusGraph](http://docs.janusgraph.org/latest/g
 
 JanusGraph is a project under [The Linux
 Foundation](https://www.linux.com/blog/Linux-Foundation-welcomes-JanusGraph),
-and includes participants from Expero, Google, GRAKN.AI, Hortonworks, and IBM.
+and includes participants from Expero, Google, GRAKN.AI, Hortonworks, IBM and Amazon.
```

Comment:
Perhaps what we should do is call out all orgs by name in the AUTHORS.md list and we update periodically (https://github.com/JanusGraph/janusgraph/blob/master/AUTHORS.txt):
"and includes participants from Amazon, Aurelius, DataStax, Expero, Google, Orion Health, IBM and other individual authors."
I think the list of individuals has the potential to get unmanageably large in the future, but the list of organizations will probably be a reasonable size.